script/cibuild

```diff
@@ -29,3 +29,6 @@ pushd t >/dev/null
   VERBOSE_LOGS=1 make X="$X" clean
   VERBOSE_LOGS=1 make X="$X" PROVE="$PROVE" PROVE_EXTRA_ARGS="$PROVE_EXTRA_ARGS"
 popd >/dev/null
+
+echo "Looking for trailing whitespace..."
+! git grep -lP '\s+$' | grep -vE '(^vendor/|\.git/objects/)'
```

Comment:
Should we use `git ls-files` here instead? I suppose that this is introducing the assumption that we're inside of a Git repository, but we make that assumption in a number of places throughout the Makefile, so I think that it is safe to do that here, too.

This carries the additional benefit that we wouldn't be grubbing through files that have been generated as a part of a previous step in script/cibuild, Makefile, or similar.

Reply:
Oh my, please ignore this comment. I glossed over `git` and thought that this was a recursive grep over the filesystem. `git grep` is certainly what we want here. Thanks!~